README.md

```diff
@@ -10,23 +10,7 @@ Create a command center for receiving and managing the grant nomination process
 
 The backend application is using the PERN Stack (Postgres, Express React, Node). The backend application is using [Express](https://expressjs.com/) Framework. The frontend application is using the [React](https://reactjs.org/) framework and several other third party libraries, bootstrapped via the [create-react-app](https://github.com/facebook/create-react-app) tool. [Postgres](https://www.postgresql.org/) is the database for the application.
 
-[Sequelize](https://sequelize.org/master/manual/migrations.html) is the ORM. This is the preferred way of communicating with the database, so the tools provided should be leveraged to help you build models and schema migrations. In particular, creation of new models should use the Sequelize CLI, which will create both a model file and a migration file.
-
-- Generate Model and Migration
-
-        npx sequelize-cli model:generate --name User --attributes firstName:string,lastName:string,email:string
-
-- Running Migration
-
-        npx sequelize-cli db:migrate
-
-- Create Seed
-
-        npx sequelize-cli seed:generate --name demo-user
-
-- Run Seed
-
-        npx sequelize-cli db:seed:all
+[Sequelize](https://sequelize.org/master/manual/migrations.html) is the ORM. This is the preferred way of communicating with the database, so the tools provided should be leveraged to help you build models and schema migrations. In particular, creation of new models should use the Sequelize CLI, which will create both a model file and a migration file. [Quick overview of commands](https://github.com/the-difference-engine/ksf/tree/develop/sequelize-cli_shortcut.md)
```

Comment:
Sorry to be weirdly picky, but I might actually move this to the bottom with the other Sequelize stuff, so it's all in one place.